### PR TITLE
improve documentation for YARP installation for macOS

### PR DESCRIPTION
**Pull request by claudia-lat**
Worked on issue https://github.com/robotology/yarp/issues/848.
<a href='#crh-start'></a><a href='#crh-data-%7B%22processed%22%3A%20%5B%22https%3A//github.com/robotology/yarp/pull/856%23issuecomment-237807020%22%2C%20%22https%3A//github.com/robotology/yarp/pull/856%23discussion_r73669094%22%2C%20%22https%3A//github.com/robotology/yarp/pull/856%23issuecomment-237813185%22%5D%2C%20%22comments%22%3A%20%7B%22General%20Comment%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/robotology/yarp/pull/856%23issuecomment-237807020%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22%5Cud83d%5Cudc4d%20%5Cr%5CnFor%20me%20is%20ok%22%2C%20%22created_at%22%3A%20%222016-08-05T09%3A36%3A03Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/2189180%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/francesco-romano%22%7D%7D%2C%20%7B%22body%22%3A%20%22Seems%20fine%20to%20me%20as%20well%21%22%2C%20%22created_at%22%3A%20%222016-08-05T10%3A05%3A28Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/14168673%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/claudiofantacci%22%7D%7D%5D%2C%20%22title%22%3A%20%22General%20Comment%22%7D%2C%20%22Pull%205a713a375aa8d3fb31449f58ebe75e219c09f55b%20doc/installation/install_mac.dox%20142%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/robotology/yarp/pull/856%23discussion_r73669094%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22Little%20style%20comment%3A%20Lua%2C%20Python%2C%20Java%20are%20not%20acronyms%2C%20and%20so%20typically%20are%20not%20capitalized.%22%2C%20%22created_at%22%3A%20%222016-08-05T09%3A56%3A38Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/1857049%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/traversaro%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20doc/installation/install_mac.dox%3AL94-354%22%7D%7D%7D'></a>
<a href='https://www.codereviewhub.com/'></a>
- [ ] <a href='#crh-comment-General Comment'></a> &nbsp;<b><a href='https://github.com/robotology/yarp/pull/856#issuecomment-237807020'>General Comment</a></b>
- <a href='https://github.com/francesco-romano'></a> 👍
For me is ok
- <a href='https://github.com/claudiofantacci'></a> Seems fine to me as well!
- [ ] <a href='#crh-comment-Pull 5a713a375aa8d3fb31449f58ebe75e219c09f55b doc/installation/install_mac.dox 142'></a> &nbsp;<b><a href='https://github.com/robotology/yarp/pull/856#discussion_r73669094'>File: doc/installation/install_mac.dox:L94-354</a></b>
- <a href='https://github.com/traversaro'></a> Little style comment: Lua, Python, Java are not acronyms, and so typically are not capitalized.

<a href='https://www.codereviewhub.com/robotology/yarp/pull/856?mark_as_completed=1'></a>&nbsp;<a href='https://www.codereviewhub.com/robotology/yarp/pull/856?approve=1'></a>&nbsp;<a href='https://github.com/robotology/yarp/pull/856'></a>
<a href='#crh-end'></a>

### CODE REVIEW

**Comment by drdanz** [comment]
This pull request is now managed with CodeReviewHub. <a href='https://github.com/robotology/yarp/pull/856'>Refresh</a>.<a href='#crh-update-none'></a>

**Comment by francesco-romano** [comment]
👍 
For me is ok

File: doc/installation/install_mac.dox [hunk 1]
```diff
@@ -80,218 +94,260 @@ The following options are also available:
...
+ brew install yarp --only-dependencies
+ \endcode
 
-\code
-brew install yarp --only-dependencies
-\endcode
- 
 
-For compiling the required bindings (LUA, JAVA, PYTHON etc) you need <b>swig</b>
+ For compiling the required bindings (LUA, JAVA, PYTHON etc) you need <b>swig</b>
```
Comment (traversaro) [inline doc/installation/install_mac.dox]: Little style comment: Lua, Python, Java are not acronyms, and so typically are not capitalized.

**Comment by claudiofantacci** [comment]
Seems fine to me as well!

**Comment by drdanz** [comment]
Merged. Thanks!

**Pull request merged**